README.md

```diff
@@ -306,6 +306,31 @@ The file containing any policy rule implementations to be used with the _evaluat
 |---------------------|-----------|---------------------------------------|
 | Kotlin script (DSL) | Evaluator | `$ORT_CONFIG_DIR/evaluator.rules.kts` |
 
+### Protecting environment variables
+
+In order to do its analysis, ORT invokes a number of external tools, such as package managers or scanners. Especially
+when interacting with package managers to obtain the dependencies of the analyzed project, this can lead to the
+execution of code in build scripts from potentially unknown sources. A possible risk in this constellation is that
+untrusted code could read sensitive information from environment variables used for the ORT configuration, such as
+database connection strings or service credentials. This is because the environment variables of a process are by
+default propagated to the child processes spawned by it.
+
+To reduce this risk, ORT filters out certain environment variables when it runs external tools in child processes.
+This filter mechanism can be configured via the following properties in the
+[ORT configuration file](./model/src/main/resources/reference.yml):
+
+| Property | Description |
+|----------|-------------|
+| deniedProcessEnvironmentVariablesSubstrings | A list of substrings that identify variables containing sensitive information. All variables that contain at least one of these strings (ignoring case) are not propagated to childprocesses. The default for this property contains strings like "PASS", "PWD", or "TOKEN", which are typically used to reference credentials. |
```

Comment:
Did you intentionally change "childprocesses" to be a single word now? Because I believe it should stay two words.

Reply:
No, this happened when fiddling with line wrapping. Sorry.